README.md

```diff
@@ -304,18 +338,20 @@ How many members should go in each level? What distribution should we be aiming
 
 No community is the same, but here's a reasonable starting point for many communities. Roughly speaking, each level is 10 times as large as the previous one.
 
-| Orbit Level     | Size | Percentage |
-| --------------- | ---- | ---------- |
-| 1 - Ambassadors | 10   | ~1%        |
-| 2 - Fans        | 100  | ~10%       |
-| 3 - Users       | 1000 | ~90%       |
-| 4 - Observers   | ∞    |            |
+| Orbit Level      | Size | Percentage |
+| ---------------  | ---- | ---------- |
+| 1 - Core         | 10   | ~1%        |
+| 2 - Contributors | 100  | ~10%       |
+| 3 - Participants | 1000 | ~90%       |
+| 4 - Observers    | ∞    |            |
 
 Observers play a mostly passive, audience-style role in the community, so the size of the level is not bounded.
 
-There are a few motivations for having targets around the distribution of orbit levels. First, it's a way to make sure the community is getting all the different contribution types that it needs. For example, when an Ambassador posts a question on the forum, Fans can help start the conversation, Users can continue it, and Observers can read it. Second, there needs to be enough mentorship available from members at lower levels to attract and transfer knowledge to members who are advancing. One Ambassador would have a hard time mentoring and supporting 50 Fans all by themself, but 5 Ambassadors might be enough.
+There are a few motivations for having targets around the distribution of orbit levels. First, it's a way to make sure the community is getting all the different contribution types that it needs. For example, when a Core member posts a question on the forum, Contributors can help start the conversation, when can continue it, and Observers can read it. 
+
+Second, there needs to be enough mentorship available from members at lower levels to attract and transfer knowledge to members who are advancing. One Core member would have a hard time mentoring and supporting 50 Contributors all by themself, but 5 Core members might be enough.
```

Comment:
The 1-to-5 ratio here conflicts with the 1-to-10 ratio in the previous paragraph

Reply:
I think we talked about this on Friday but I'm okay if we want to take out all the verbiage around orbit level distributions.